guides/common/modules/proc_converting-hosts-with-convert2rhel.adoc

```diff
@@ -0,0 +1,23 @@
+[id="converting-hosts-with-convert2rhel_{context}"]
+
+= Converting Hosts with Convert2RHEL
+
+Use the following procedure to perform the conversion.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* *>* *All Hosts* and select the target hosts you want to convert.
+You can use the *Search* field to filter the host list.
+. From the *Select Action* list, select Schedule Remote Job.
+. On the *Job Invocation* page, define the main job settings.
+. In the *Job category* field, select Convert 2 RHEL.
+. Optional: In the *Job template* field, select Convert to RHEL.
```

Comment:
This is not optional, it gets autoselected but must be selected at the end. Perhaps the step could be "Verify the *Job template* field is set to Covnert to RHEL" or similar

Reply:
Updated.